docs/exposing-binding-data/detect-bindings.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_position: 6
+---
+
+# Detect Bindings through owned resources
+
+The Service Bindings operator can detect binding from resources owned by the
+service custom resource.  Service Binding operator can detect Routes, Services,
+ConfigMaps, and Secrets owned by the backing service custom resource.
+Annotation is not required to extract the binding data.
+
+This option is initiated by the setting `detectBindingResources: true` in the
```

Comment:
Re-write with this content procedure:

- Set this `detectBindingResources` API option to `true` in the `ServiceBinding` custom resource.